src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.PathologyRecordFixtures.generatePathologyRecord;
+
+@ExtendWith(MockitoExtension.class)
+public class BundleMapperTest {
+
+    @Test
+    public void testMapMessageToPathologyRecord() {
+        final BundleMapper bundleMapper = new BundleMapper();
+
+        final Bundle bundle = bundleMapper.mapToBundle(generatePathologyRecord());
+
+        assertNotNull(bundle.getMeta().getLastUpdated());
+        assertEquals(bundle.getMeta().getProfile().get(0).asStringValue(), "https://fhir.nhs.uk/STU3/StructureDefinition/ITK-Message-Bundle-1");
+        assertThat(bundle.getIdentifier().getSystem()).isEqualTo("https://tools.ietf.org/html/rfc4122");
+        assertNotNull(bundle.getIdentifier().getValue());
+        assertEquals(bundle.getType(), Bundle.BundleType.MESSAGE);
+
+        final int EXPECTED_NUMBER_OF_RESOURCES = 3;
+        assertEquals(EXPECTED_NUMBER_OF_RESOURCES, bundle.getEntry().size());
+
+        Bundle.BundleEntryComponent bundleEntryComponentForPerformerResource = bundle.getEntry().get(0);
+        assertNotNull(bundleEntryComponentForPerformerResource.getFullUrl());
```

Comment:
Can we assert the url? I think we're setting this in the `BundleMapper`

Reply:
Since the UUID is randomly generated, it's hard to test that. Best we can do is just assert that it's not null/empty

Reply:
Oh yes, we're generating a UUID now. That's fair enough then

Reply:
@andrewwa-kainos pointed out we could inject this instead (rather than make it static). So we can test this now